inst/rstudio/templates/project/resources/common/index.Rmd

```diff
@@ -0,0 +1,55 @@
+--- 
+title: "A Minimal Book Example"
+author: "John Doe"
+date: "`r Sys.Date()`"
+site: bookdown::bookdown_site
+url: https://bookdown.org/johndoe/bookdown-demo
+cover-image: cover.png
```

Comment:
We have a copy of the cover image here: https://github.com/rstudio/bookdown/blob/master/inst/examples/images/cover.jpg so I tend not to duplicate it here.

```suggestion
cover-image: https://bookdown.org/yihui/bookdown/images/cover.jpg
```

Reply:
The idea here is to have a template project to start with. 

I am fine with a link here to show that this is possible too, but in our book we usually use a cover image that is in the book, and that gets preprended with the url. See
 * [R Markown Cookbook](https://github.com/yihui/rmarkdown-cookbook/blob/b7d818ccd5ff569194a67d1462a94f85716f5fa8/index.Rmd#L20)
 * [Bookdown Book](https://github.com/rstudio/bookdown/blob/b3f11ba9daed638f3a71e1fecd6c2e626f80d355/inst/examples/index.Rmd#L18)
 * [Definitive Guide](https://github.com/rstudio/rmarkdown-book/blob/ed81b894d5d2665d6ef4657c5840634512b97ff5/index.Rmd#L21)

That is why I thought showing the common practice was a good idea. 

We could maybe not include it and just add a paragraph on how to add a cover image using this field ?

Reply:
> We could maybe not include it and just add a paragraph on how to add a cover image using this field ?

Yes, that's what I prefer.

Reply:
Could we re-visit this? Can we make the image super small? This is a tricky one (because you have to use `cover-image` PLUS `url`) for it to work properly, and easier to see with a working example than to read.

Reply:
Yes, we could use the same approach as https://github.com/rstudio/bookdown/pull/1201#discussion_r675653417.

Reply:
* Retrieve a file from within the package instead of creating a new one.

Reply:
We now retrieve the logo. But this is still not great IMO. I'll explain in main thread